troubleshooting/known_issues/iuf_waits_for_ten_mins.md

```diff
@@ -0,0 +1,96 @@
+# IUF waits for 10 minutes to get workflow status before executing
+
+## Issue Description
+
+When running IUF commands, the `iuf-cli` waits for 10 minutes before starting execution. During this time, it repeatedly displays a warning message: "Unable to get workflow status."
+
+## Error Identification
+
+While executing IUF commands the following warning is displayed by `iuf-cli` for 10 minutes .
+
+```sh
+ncn-m001:~ # iuf -a "${ACTIVITY_NAME}" -m "${MEDIA_DIR}" run --site-vars "${ADMIN_DIR}/site_vars.yaml" -bpcd "${ADMIN_DIR}" -r management-nodes-rollout --limit-management-rollout ncn-w003
+INFO All logs will be stored in /etc/cray/upgrade/csm/iuf/install-products/log/20250306145455
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+WARN Unable to get workflow status. Retrying after 10 seconds...
+INFO [ACTIVITY: install-products                               ] BEG Install started at 2025-03-06 14:54:55.095146
+INFO [IUF SESSION: install-products-h16zj                      ] BEG Started at 2025-03-06 15:04:15.305532
+INFO [STAGE: management-nodes-rollout                          ] BEG Argo workflow: install-products-h16zj-management-nodes-rollout-vg48w 
+```
+
+## Error Conditions
+
+If an IUF session is abruptly terminated (e.g., using Ctrl+C), the running workflow is also terminated.  Although IUF stores workflow data in a state file `(activity_dict.yaml)`, the termination causes the workflow status to become "Unknown."
+
+When IUF attempts to retrieve the workflow status, it fails because the workflow no longer exists. This results in a discrepancy between the activity data stored by IUF and the `argo` workflow server.
+
+## Workaround Description
+
+To resolve this issue, follow these steps:
+
+Step 1: Locate the `activity_dict.yaml` file in the state directory of the activity.
```

Comment:
Number this 1. and then indent the lines under it. This should be done for all of the "steps". It will be easier to follow if they are numbered.